policies/otc-policies.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--#include virtual="/inc/head.shtml" -->
+
+<body>
+<!--#include virtual="/inc/banner.shtml" -->
+
+<div id="main">
+  <div id="content">
+    <div class="blog-index">
+      <article>
+        <header>
+          <h2>OpenSSL Technical Policies</h2>
+          <h5>
+            First issued 30th September 2020<br/>
+            Last modified 30th September 2020
+          </h5>
+        </header>
+
+        <div class="entry-content">
+
+          <p>This document lists the technical policies and procedures established
+          by the OTC based on the <a href="omc-bylaws.html">project bylaws</a>
+          and the requirements specified by the OMC.</p>
+
+          <h3><a name="voting-procedure">Voting Procedure</a></h3>
+
+          The following implementary regulations complement the
+          <a href="omc-bylaws.html#otc-voting">OTC Voting Procedures</a>
+          stated in the project bylaws:
+
+          <p>The proposer of a vote is ultimately responsible for updating the votes.txt
+          file in the repository.  Outside of a face to face meeting, voters MUST reply
```

Comment:
Can this be the place where the repository is advertised?

Reply:
Good point. How about like this: 30e850b5fc8d58a527d04c8ae2dd4bde0d98c476 ?